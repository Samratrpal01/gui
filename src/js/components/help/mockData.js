export const helpProps = {
  docsVersion: undefined,
  isHosted: false,
  menderVersion: 'master',
  isEnterprise: true,
  tenantCapabilities: {},
  tokens: [],
  versions: {
    repos: {
      mender: '3.3.0',
      'mender-artifact': '3.8.0',
      'mender-binary-delta': '1.3.1',
      'mender-cli': '1.8.0',
      'mender-configure-module': '1.0.4',
      'mender-connect': '2.0.2',
      'mender-convert': '3.0.0',
      'mender-gateway': '1.0.0',
      'monitor-client': '1.2.0'
    },
    releaseDate: '01-01-2003'
  }
};
