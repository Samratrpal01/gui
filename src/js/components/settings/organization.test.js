import React from 'react';
import { Provider } from 'react-redux';
import thunk from 'redux-thunk';
import configureStore from 'redux-mock-store';
import MyOrganization, {
  OrgHeader,
  TrialExpirationNote,
  DeviceLimitExpansionNotification,
  CancelSubscriptionAlert,
  CancelSubscriptionButton
} from './organization';
import { defaultState, undefineds } from '../../../../tests/mockData';
import { render } from '../../../../tests/setupTests';

const mockStore = configureStore([thunk]);

describe('MyOrganization Component', () => {
  let store;
  beforeEach(() => {
    Date.now = jest.fn(() => new Date('2020-07-01T12:00:00.000Z'));
    store = mockStore({
      ...defaultState,
      app: {
        ...defaultState.app,
        features: {
          ...defaultState.app.features,
          isHosted: true
        }
      },
      devices: {
        ...defaultState.devices,
        byStatus: {
          ...defaultState.devices.byStatus,
          accepted: {
            ...defaultState.devices.byStatus.accepted,
            total: 1
          }
        },
        limit: null
      },
      organization: {
        ...defaultState.organization,
        card: {
          last4: '1234',
          expiration: { month: 8, year: 1230 },
          brand: 'Visa'
        },
        organization: {
          ...defaultState.organization.organization,
          plan: 'enterprise',
          tenant_token: 'test',
          trial: true,
          trial_expiration: new Date('2021-01-01T00:00:00Z')
        }
      }
    });
  });

  it('renders correctly', async () => {
    const { baseElement } = render(
      <Provider store={store}>
        <MyOrganization />
      </Provider>
    );
    const view = baseElement.firstChild.firstChild;
    expect(view).toMatchSnapshot();
    expect(view).toEqual(expect.not.stringMatching(undefineds));
  });
});

describe('smaller components', () => {
  [OrgHeader, TrialExpirationNote, DeviceLimitExpansionNotification, CancelSubscriptionAlert, CancelSubscriptionButton].forEach(Component => {
    it(`renders ${Component.displayName || Component.name} correctly`, () => {
      const { baseElement } = render(
        <Component
          trial_expiration="2019-10-05T13:00:00.000Z"
          isTrial={true}
          handleCancelSubscription={jest.fn}
          orgName="test"
          mailBodyTexts={{ billing: 'bill this', upgrade: 'upgrade here' }}
        />
      );
      const view = baseElement.firstChild.firstChild;
      expect(view).toMatchSnapshot();
      expect(view).toEqual(expect.not.stringMatching(undefineds));
    });
  });
});
