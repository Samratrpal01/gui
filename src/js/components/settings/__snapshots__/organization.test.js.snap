// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyOrganization Component renders correctly 1`] = `
.emotion-0 {
  list-style: none;
  margin: 0;
  padding: 0;
  position: relative;
  padding-top: 8px;
  padding-bottom: 8px;
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -ms-flex-pack: start;
  -webkit-justify-content: flex-start;
  justify-content: flex-start;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  position: relative;
  -webkit-text-decoration: none;
  text-decoration: none;
  width: 100%;
  box-sizing: border-box;
  text-align: left;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 16px;
  padding-right: 16px;
}

.emotion-1.Mui-focusVisible {
  background-color: rgba(0, 0, 0, 0.12);
}

.emotion-1.Mui-selected {
  background-color: rgba(25, 118, 210, 0.08);
}

.emotion-1.Mui-selected.Mui-focusVisible {
  background-color: rgba(25, 118, 210, 0.2);
}

.emotion-1.Mui-disabled {
  opacity: 0.38;
}

.emotion-2 {
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
  min-width: 0;
  margin-top: 6px;
  margin-bottom: 6px;
}

.emotion-3 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1rem;
  line-height: 1.5;
  letter-spacing: 0.00938em;
  display: block;
}

.emotion-4 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 0.875rem;
  line-height: 1.43;
  letter-spacing: 0.01071em;
  color: rgba(0, 0, 0, 0.6);
  display: block;
}

.emotion-5 {
  margin: 0;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  border-width: 0;
  border-style: solid;
  border-color: rgba(0, 0, 0, 0.12);
  border-bottom-width: thin;
}

.emotion-9 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.5rem;
}

.emotion-11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -ms-flex-pack: start;
  -webkit-justify-content: flex-start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  position: relative;
  -webkit-text-decoration: none;
  text-decoration: none;
  width: 100%;
  box-sizing: border-box;
  text-align: left;
  padding-top: 8px;
  padding-bottom: 8px;
}

.emotion-11.Mui-focusVisible {
  background-color: rgba(0, 0, 0, 0.12);
}

.emotion-11.Mui-selected {
  background-color: rgba(25, 118, 210, 0.08);
}

.emotion-11.Mui-selected.Mui-focusVisible {
  background-color: rgba(25, 118, 210, 0.2);
}

.emotion-11.Mui-disabled {
  opacity: 0.38;
}

.emotion-12 {
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
  min-width: 0;
  margin-top: 4px;
  margin-bottom: 4px;
}

.emotion-15 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 0.875rem;
  line-height: 1.75;
  letter-spacing: 0.02857em;
  text-transform: uppercase;
  min-width: 64px;
  padding: 6px 8px;
  border-radius: 4px;
  -webkit-transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  color: #1976d2;
}

.emotion-15::-moz-focus-inner {
  border-style: none;
}

.emotion-15.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-15 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-15:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: rgba(25, 118, 210, 0.04);
}

@media (hover: none) {
  .emotion-15:hover {
    background-color: transparent;
  }
}

.emotion-15.Mui-disabled {
  color: rgba(0, 0, 0, 0.26);
}

.emotion-16 {
  display: inherit;
  margin-right: 8px;
  margin-left: -4px;
}

.emotion-16>*:nth-of-type(1) {
  font-size: 20px;
}

.emotion-18 {
  overflow: hidden;
  pointer-events: none;
  position: absolute;
  z-index: 0;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  border-radius: inherit;
}

.emotion-24 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.25rem;
}

<div
  class="margin-top-small"
>
  <h2
    class="margin-top-small"
  >
    Organization and billing
  </h2>
  <ul
    class="MuiList-root MuiList-padding emotion-0"
  >
    <div
      class="margin-top-small org-settings-item"
    >
      <li
        class="flexbox column MuiListItem-root MuiListItem-gutters MuiListItem-padding Mui-disabled MuiListItem-alignItemsFlexStart emotion-1"
        disabled=""
      >
        <div
          class="MuiListItemText-root MuiListItemText-multiline emotion-2"
          style="display: grid; width: 100%; margin-bottom: 8px;"
        >
          <span
            class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary emotion-3"
          >
            Organization name
          </span>
          <div
            class="MuiTypography-root MuiTypography-body2 two-columns MuiListItemText-secondary emotion-4"
            style="display: grid; width: 100%; margin-bottom: 8px;"
          >
            <div>
              test
            </div>
            <a
              class="align-right"
            />
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth emotion-5"
          style="width: 500px;"
        />
      </li>
    </div>
    <div
      class="margin-top-small org-settings-item"
    >
      <li
        class="flexbox column MuiListItem-root MuiListItem-gutters MuiListItem-padding Mui-disabled MuiListItem-alignItemsFlexStart emotion-1"
        disabled=""
      >
        <div
          class="MuiListItemText-root MuiListItemText-multiline emotion-2"
          style="display: grid; width: 100%; margin-bottom: 0px;"
        >
          <span
            class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary emotion-3"
          >
            <div
              class="flexbox center-aligned"
            >
              <div
                style="padding-right: 10px;"
              >
                Organization token
              </div>
              <div
                class=""
                data-mui-internal-clone-element="true"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-9"
                  data-testid="InfoIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
                  />
                </svg>
              </div>
            </div>
          </span>
          <div
            class="MuiTypography-root MuiTypography-body2 MuiListItemText-secondary emotion-4"
            style="display: grid; width: 100%; margin-bottom: 0px;"
          >
            <div
              style="width: 468px;"
            >
              <div
                class="attributes MuiListItem-root MuiListItem-padding Mui-disabled emotion-11"
                disabled=""
              >
                <div
                  class="MuiListItemText-root emotion-12"
                >
                  <div
                    class="MuiTypography-root MuiTypography-body2 MuiListItemText-secondary emotion-4"
                    title="test"
                  >
                    <div>
                      <span
                        class="break-all inventory-text tenant-token-text"
                      >
                        test
                      </span>
                       
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth emotion-5"
          style="width: 500px;"
        />
      </li>
      <div>
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  emotion-15"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-startIcon MuiButton-iconSizeMedium emotion-16"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-9"
              data-testid="FileCopyIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm-1 4 6 6v10c0 1.1-.9 2-2 2H7.99C6.89 23 6 22.1 6 21l.01-14c0-1.1.89-2 1.99-2h7zm-1 7h5.5L14 6.5V12z"
              />
            </svg>
          </span>
          Copy to clipboard
          <span
            class="MuiTouchRipple-root emotion-18"
          />
        </button>
        <div
          style="height: 30px; padding: 15px;"
        />
      </div>
    </div>
    <div
      class="margin-top-small org-settings-item"
    >
      <li
        class="flexbox column MuiListItem-root MuiListItem-gutters MuiListItem-padding Mui-disabled MuiListItem-alignItemsFlexStart emotion-1"
        disabled=""
      >
        <div
          class="MuiListItemText-root MuiListItemText-multiline emotion-2"
          style="display: grid; width: 100%; margin-bottom: 8px;"
        >
          <span
            class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary emotion-3"
          >
            Current plan
          </span>
          <div
            class="MuiTypography-root MuiTypography-body2 two-columns MuiListItemText-secondary emotion-4"
            style="display: grid; width: 100%; margin-bottom: 8px;"
          >
            <div>
              Trial
            </div>
            <a
              class="align-right"
              href="https://mender.io/plans/pricing"
              rel="noopener noreferrer"
              target="_blank"
            >
              Compare product plans
            </a>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth emotion-5"
          style="width: 500px;"
        />
        <div
          class="flexbox centered text-muted"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall emotion-24"
            data-testid="ErrorIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z"
            />
          </svg>
          <span
            class="margin-left-small"
          >
            Your trial expires in 
            6 months
            . 
            <a
              href="/settings/upgrade"
            >
              Upgrade to a paid plan
            </a>
          </span>
        </div>
      </li>
    </div>
    <div
      class="margin-top-small org-settings-item"
    >
      <li
        class="flexbox column MuiListItem-root MuiListItem-gutters MuiListItem-padding Mui-disabled MuiListItem-alignItemsFlexStart emotion-1"
        disabled=""
      >
        <div
          class="MuiListItemText-root MuiListItemText-multiline emotion-2"
          style="display: grid; width: 100%; margin-bottom: 8px;"
        >
          <span
            class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary emotion-3"
          >
            Current add-ons
          </span>
          <div
            class="MuiTypography-root MuiTypography-body2 two-columns MuiListItemText-secondary emotion-4"
            style="display: grid; width: 100%; margin-bottom: 8px;"
          >
            <div>
              You currently don't have any add-ons
            </div>
            <a
              class="align-right"
            >
              Purchase an add-on
            </a>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth emotion-5"
          style="width: 500px;"
        />
        <div
          class="flexbox centered text-muted"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall emotion-24"
            data-testid="ErrorIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z"
            />
          </svg>
          <span
            class="margin-left-small"
          >
            Your trial expires in 
            6 months
            . 
            <a
              href="/settings/upgrade"
            >
              Upgrade to a paid plan
            </a>
          </span>
        </div>
      </li>
      <div
        class="margin-left-small margin-bottom"
      >
        <a
          class="flexbox center-aligned"
          href="https://mender.io/plans/pricing"
          rel="noopener noreferrer"
          target="_blank"
        >
          <div
            style="max-width: 200px;"
          >
            Compare plans and add-ons at mender.io
          </div>
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall emotion-24"
            data-testid="OpenInNewIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
            />
          </svg>
        </a>
      </div>
    </div>
  </ul>
  <p
    class="margin-left-small margin-right-small"
    style="max-width: 500px;"
  >
    <a
      href=""
    >
      End trial
       and deactivate account
    </a>
  </p>
</div>
`;

exports[`smaller components renders CancelSubscriptionAlert correctly 1`] = `
<div
  class="margin-top-large alert alert-error"
  style="max-width: 500px;"
>
  <p>
    We've started the process to cancel your plan and deactivate your account.
  </p>
  <p>
    We'll send you an email confirming your deactivation. If you have any question at all, contact us at
     
    <strong>
      <a
        href="mailto:support@mender.io"
      >
        support@mender.io
      </a>
    </strong>
  </p>
</div>
`;

exports[`smaller components renders CancelSubscriptionButton correctly 1`] = `
<p
  class="margin-left-small margin-right-small"
  style="max-width: 500px;"
>
  <a
    href=""
  >
    End trial
     and deactivate account
  </a>
</p>
`;

exports[`smaller components renders DeviceLimitExpansionNotification correctly 1`] = `
.emotion-0 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.25rem;
}

<div
  class="flexbox centered text-muted"
>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall emotion-0"
    data-testid="ErrorIcon"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z"
    />
  </svg>
  <span
    class="margin-left-small"
  >
    To increase your device limit,
     
    <a
      href="/settings/upgrade"
    >
      upgrade to a paid plan
    </a>
    .
  </span>
</div>
`;

exports[`smaller components renders OrgHeader correctly 1`] = `
.emotion-0 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.5rem;
}

<div
  class="flexbox center-aligned"
>
  <div
    style="padding-right: 10px;"
  >
    Organization token
  </div>
  <div
    class=""
    data-mui-internal-clone-element="true"
  >
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-0"
      data-testid="InfoIcon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
      />
    </svg>
  </div>
</div>
`;

exports[`smaller components renders TrialExpirationNote correctly 1`] = `
.emotion-0 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.25rem;
}

<div
  class="flexbox centered text-muted"
>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall emotion-0"
    data-testid="ErrorIcon"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z"
    />
  </svg>
  <span
    class="margin-left-small"
  >
    Your trial expires in 
    9 months
    . 
    <a
      href="/settings/upgrade"
    >
      Upgrade to a paid plan
    </a>
  </span>
</div>
`;
